pom.xml

```diff
@@ -25,7 +25,7 @@
     <useBeta>true</useBeta>
 
     <!-- Jenkins Plug-in Dependencies Versions -->
-    <checks-api.version>1.0.2</checks-api.version>
+    <checks-api.version>2.0.0-SNAPSHOT</checks-api.version>
```

Comment:
It would make sense to use the associated incrementals build:
https://repo.jenkins-ci.org/incrementals/io/jenkins/plugins/checks-api/